exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,15 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+
+function upCase(name){
```

Comment:
For this exercise, you want a nested function. A nested function is a function inside another function. In this case, you want one function called shout which contains the upCase function you have created. The upCase function would be a nested function if used in this way.